File: java/8.5/example7.java
import java.util.Scanner;

public class example7 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter your age");
        Integer age = Integer.parseInt(scanner.nextLine());
        if (age < 14) {
            System.out.println("You are not old enough");
        } else if (age < 16) {
            System.out.println("You need an adult present");
        }
    }
}

File: rust/src/8.5/example7.rs
fn main() {
    let mut ages = String::new();
    println!("Enter your age");
    std::io::stdin().read_line(&mut ages).unwrap();
    let age = ages.trim().parse::<i32>().unwrap();
    if age < 14 {
        println!("You are not old enough");
    } else if age < 16 {
        println!("You need an adult present.")
    }
}